portal-web/docroot/html/taglib/ui/input_permissions/page.jsp

```diff
@@ -119,11 +119,15 @@ String modelName = (String)request.getAttribute("liferay-ui:input-permissions:mo
 					<option <%= inputPermissionsViewRole.equals(RoleConstants.OWNER) ? "selected=\"selected\"" : "" %> value="<%= RoleConstants.OWNER %>"><liferay-ui:message key="owner" /></option>
 				</select>
 
-				<div class="mt-3 <%= inputPermissionsShowOptions ? "hide" : "" %>" id="<%= uniqueNamespace %>inputPermissionsShowOptionsLink" style="margin-bottom: 60px;">
-					<a class="btn btn-secondary btn-sm mt-3" href="javascript:<%= uniqueNamespace %>inputPermissionsShowOptions();" role="button"><liferay-ui:message key="more-options" /></a> <liferay-ui:icon-help message="input-permissions-more-options-help" />
+				<div>
+					<button aria-controls="<%= uniqueNamespace %>inputPermissionsTable" aria-expanded="<%= inputPermissionsShowOptions %>" class="btn btn-secondary btn-sm mt-3" id="<%= uniqueNamespace %>inputPermissionsOptionsButton" onclick="<%= uniqueNamespace %>inputPermissionsToggle();" type="button" value="<%= LanguageUtil.get(request, "more-options") %>">
+						<liferay-ui:message key="more-options" />
```

Comment:
In this button now we don't check the `inputPermissionsShowOptions` value like before (you are just checking for the `aria-expanded` attribute), so always the text will be "more options" at the first moment. The value attribute must check that `inputPermissionsShowOptions` value to show _More Options_ or _Hide Options_